src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1386,6 +1386,18 @@ function ($a) {
                                 ->children()
                                     ->scalarNode('dsn')->end()
                                     ->scalarNode('serializer')->defaultNull()->info('Service id of a custom serializer to use.')->end()
+                                    ->arrayNode('symfony_serializer')
```

Comment:
what about a less specific `context` array, eg.`context.format` to be used by SF serializer

Reply:
you mean we could have this kind of configuration?
```
messenger:
    transports:
        foo:
            context:
                format: xml
                xml_root_node_name: foo
```
and then `$context['format']` would be passsed to `$this->serializer->(de)serialize()` in `Symfony\Component\Messenger\Transport\Serialization\SerializerInterface`?